storybookjs/storybook
# Merge pull request #9383 from baker-travis/knob-groups-overflow-not-scrolling

fixes #9299

Issue:

## What I did
Allowed the overflow of the container holding the knob categories.

## How to test
1. Use the official-storybook example project
2. Navigate to Addons -> Knobs -> withKnobs -> tweaks static values organized in groups.
3. Move the panel that contains "Knobs" to the right side of your screen.
4. Observe that you can now scroll the knob categories.

- Is this testable with Jest or Chromatic screenshots? No
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.
